src/planwise/model/import_job.clj

```diff
@@ -390,6 +390,10 @@
                          :type-field type-field)]
     (import-job job-value)))
 
+(defn restore-job
+  [job]
+  (import-job (:state job) (:value job)))
```

Comment:
I mean tasks that are already executing. There can be up to `concurrent-workers` simultaneous (parallel) tasks.